### PR TITLE
Inner ring glow effect

### PR DESCRIPTION
**Pull request by Nixon506E**

### CODE REVIEW

File: src/UICircularProgressRing/UICircularRingLayer.swift [hunk 1]
```diff
@@ -215,6 +215,7 @@ class UICircularRingLayer: CAShapeLayer {
         ctx.setLineJoin(.round)
         ctx.setLineCap(ring.innerCapStyle)
         ctx.setStrokeColor(ring.innerRingColor.cgColor)
+        ctx.setShadow(offset: .zero, blur: ring.glowStrength, color: ring.innerRingColor)
```
Comment (luispadron) [inline src/UICircularProgressRing/UICircularRingLayer.swift]: should this be renamed `innerRingBlurStrength` instead?

**Comment by luispadron** [comment]
Hey there thanks for contributing! Could you add tests for this new property, that would be sweet. Also, what exactly does this look like/what is the use case for this new feature?

**Comment by Nixon506E** [comment]
I had a client request that the progress bar in the circle have a glow effect but not the ring itself so as to highlight that it is in an in-progress state and not full or empty.

**Comment by luispadron** [comment]
Hey doing some cleaning, while I appreciate you sharing this addition I don't think this entirely needed for this library right now. Thanks again for your work, it is appreciated!

**Pull request closed without merging**